### ISSUE TITLE
YOLOV3-voc.cfg

### ISSUE THREAD

**Issue by mingmmq** [pjreddie/darknet#557]
Seem the voc cfg is not provided, but mentioned in the website.

**Comment by sivagnanamn** [comment]
Will there be a tiny version of V3 network?

**Comment by czero69** [comment]
quoting @TheMikeyR user:
"There are no tiny yolov3, yolov3 is just a few improvements to yolov2.
You could try to do the suggestions by Alexey but it seems no one have managed to test it yet AlexeyAB#552 (comment) "

I trained shrinked yolo3 model as AlexeyAB suggest, and it works very fine for me

**Comment by Lunaticf** [comment]
@czero69 can you share the shrinked yolo3 model and cfg,cause I ran into some problem when I modify it and train it,and I have limited device as a student.sorry,my english is poor.Thank you very much

**Comment by czero69** [comment]
Yes, of course. Note, that all changes I did are the same as AlexeyAB#552 (comment). Note also, I have classes=6. I trained it for ripped Coco dataset (person, car, ..) +  my own sample. With this dataset, I encountered 'nans' on the console, and the average loss was high ~3.5-4.0. But the results are astonishingly good. I leave defaults anchor boxes defined for Coco set.
[yolov3-pedestrians_coco_subset.cfg.zip](https://github.com/pjreddie/darknet/files/1910602/yolov3-pedestrians_coco_subset.cfg.zip)
